certificate-service/pdfbox-generator/src/main/java/se/inera/intyg/certificateservice/pdfboxgenerator/FK7211PdfGenerator.java

```diff
@@ -35,186 +22,56 @@ public class FK7211PdfGenerator {
       "form1[0].#subform[0].ksr_kryssruta[1]";
   public static final String QUESTION_BERAKNAT_NEDKOMSTDATUM_CERTIFIER_NURSE_FIELD_ID =
       "form1[0].#subform[0].ksr_kryssruta[2]";
-  public static final String SIGNATURE_DATE_FIELD_ID =
-      "form1[0].#subform[0].flt_datUnderskrift[0]";
-  public static final float SIGNATURE_SIGNATURE_POSITION_X = 171;
-  public static final float SIGNATURE_SIGNATURE_POSITION_Y = 522;
-  public static final String SIGNATURE_FULL_NAME_FIELD_ID =
-      "form1[0].#subform[0].flt_txtNamnfortydligande[0]";
-  public static final String SIGNATURE_PA_TITLE_FIELD_ID =
-      "form1[0].#subform[0].flt_txtBefattning[0]";
-  public static final String SIGNATURE_SPECIALITY_FIELD_ID =
-      "form1[0].#subform[0].flt_txtEventuellSpecialistkompetens[0]";
-  public static final String SIGNATURE_HSA_ID_FIELD_ID =
-      "form1[0].#subform[0].flt_txtLakarensHSA-ID[0]";
-  public static final String SIGNATURE_WORKPLACE_CODE_FIELD_ID =
-      "form1[0].#subform[0].flt_txtArbetsplatskod[0]";
-  public static final String SIGNATURE_CARE_UNIT_CONTACT_INFORMATION_FIELD_ID =
-      "form1[0].#subform[0].flt_txtVardgivarensNamnAdressTelefon[0]";
-  public static final String CHECKED_BOX_VALUE = "1";
-  public static final String UNCHECKED_BOX_VALUE = "Off";
-  public static final String DIGITALLY_SIGNED_TEXT =
-      "Detta är en utskrift av ett elektroniskt intyg. "
-          + "Intyget har signerats elektroniskt av intygsutfärdaren.";
-
-
-  private final PDDocument pdDocument;
-  private final PDAcroForm acroForm;
-
-  public FK7211PdfGenerator(Certificate certificate) {
-    final var template = certificate.certificateModel().pdfTemplatePath();
-    try (final var inputStream = getClass().getClassLoader().getResourceAsStream(template)) {
-      if (inputStream == null) {
-        throw new IllegalArgumentException("Template not found: " + template);
-      }
-      pdDocument = Loader.loadPDF(inputStream.readAllBytes());
-      final var documentCatalog = pdDocument.getDocumentCatalog();
-      acroForm = documentCatalog.getAcroForm();
+  public static final ElementId QUESTION_BERAKNAT_NEDKOMSTDATUM_ID = new ElementId("1");
```

Comment:
Feels a bit unnatural to copy paste in the question ids here. But were in a different package and I don't think the right way to go is to add a dependency to where these ids exists. Also, I was thinking that we could programatically match the questions filled in to the fields in the pdfs. But then we would have to add the pdf field id's to the certificate model, which might also be something we don't want to do.